CHANGELOG.txt

```diff
@@ -0,0 +1,11 @@
+Changelog
+=========
+
+0.0.17 (2017-11-28)
+-------------------
+- Correctly scale the flame graph so it shows all the samples.
+
+0.0.16 (2017-07-07)
+-------------------
+- Bug fixes for endpoints that do not perform queries nor have templates.
+
```

Comment:
You have an extra newline at the bottom here. No biggie, but you should remove it.